corehq/toggles.py

```diff
@@ -1660,6 +1660,15 @@ def _commtrackify(domain_name, toggle_is_enabled):
     relevant_environments={'icds', 'icds-new', 'india'},
 )
 
+
+ICDS_DISHA_API_V2 = StaticToggle(
+    'icds_disha_access',
```

Comment:
We already have another toggle with this name for actual DISHA. Should change this to something else `nic_indicators` or something like that.

Reply:
sure.